Beyond08-SOW.md

```diff
@@ -0,0 +1,473 @@
+# Red Hat Beyond 08
+
+Statement of Work (SOW)
+***
+
+## Pre-requirements
+
+You will be working on a project during this course.
+
+The following tech stack will be used.
+
+If you don't have knowledge/experience in any of them, it is worthwhile to look
+them up. The more you know the easier it will be.
+
+* Linux OS - able to use basic command line tools such as cd, ls, cat, grep,
+  less, vi, ...
+* Python - if/else, for loops, functions, classes
+* Django - create a basic or sample project
+* Git and GitHub - create a repo, commit, push, pull requests (PR), merge
+* HTML/CSS (Bootstrap) - design a basic web page from scratch, including using
+  CSS classes for basic formatting
+* Testing- unit tests and functional tests with pytest
+* Virtualization - Spin up a VM using Vagrant and VirtualBox
+* Databases - Basic familiarity with database concepts
+
+## Milestones
+
+### Demo 1 (November 13th): Project selection and setting up a DEV environment
+
+The main purpose of the demo is to motivate you to come together as a project
+development team and establish your shared working environment and process. As
+such, we will monitor when and how you do the work of preparing for the demo,
+and the process is as important (If not more so) as the final results you will
+present.
+
+As part of the preparation for this demo you should establish a new 
+**public** channel in the Beyond Slack workspace for communications inside 
+the team and with its mentors, get to know your teammates and your mentors 
+and come up with the basic idea and name for your project.  
+
+**Technical requirements**
+
+To be ready for the demo, your project should include the following elements:
+
+1. A working Vagrant environment that brings up the default Django webserver
+2. A basic Django application that includes a static web page showing the 
+   name of your project and a short description about it
+3. A GitHub repo in the Beyond organization with:
+    1. A `README.md` file that introduces your project
+    2. An MIT license file
+    3. The contribution guide in a `CONTRIBUTING.md` file
+    4. The code, as defined by other sections on this list
+    
+    (See below for detailed instructions about the contents of the 
+   documentation files)
+5. A basic GitHub workflow that executes Flake8 tests on files when there is a
+   PR
+
+Note that all these topics have been covered by the introduction exercise you
+have been given.
+
+**Work division**
+
+Ensure that every member of your team plays an active role in preparing for
+the demo, we require that you divide the work between you in the following way:
+
+1. Student1: vagrant
+2. Student2: bootstrap.sh
+3. Student3: Django
+4. Student4: Web page
+5. Student5: Documentation files + flake8
```

Comment:
maybe add something about sharing the information with each other later, so everyone will be able to use and maintain all of those aspects

Reply:
Ack